prime-angular-frontend/src/app/modules/adjudication/adjudication-routing.module.ts

```diff
@@ -57,6 +63,31 @@ const routes: Routes = [
             component: EnrolleeBannerPageComponent,
             data: { title: 'Enrollee Banner' }
           },
+          {
```

Comment:
Feels like we can do something to reduce the number of components.  They should have different routes, but based on the route the page component could have something slightly different displayed like `/enrolee` filters on enrollee emails.  It looks like some of these pages are almost exactly the same.